CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+## How to contribute to simplectemplate
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
+Please note we have a Pull Request Process, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+* Creating a new branch off of the develop branch
+* All changes to the code should have appropriate unit tests
+* All code should match PSR-2 style
+* Submit a pull request again develop as the base branch
+* All travis checks will need to pass
+#### **Did you find a bug?**
+
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/dafky2000/simplectemplate/issues).
+
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/dafky2000/simplectemplate/issues/new). Be sure to include a **title and clear description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+
+#### **Did you write a patch that fixes a bug?**
+
+
+* Open a new GitHub pull request with the patch.
+
+
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+
+
+#### **Do you intend to add a new feature or change an existing one?**
```

Comment:
Let's get rid of this section, I don't mind getting feature requests (as long as they aren't duplicated) as a method of getting some initial feedback.